docker/ssl/authorization.js

```diff
@@ -0,0 +1,170 @@
+var clientKey = '';
+
+// Check certificate MQTTS.
+function authenticate(s) {
+    if (!s.variables.ssl_client_s_dn || !s.variables.ssl_client_s_dn.length) {
+        s.deny();
+        return
+    }
+
+    s.on('upload', function (data) {
+        while (data == '') {
+            return s.AGAIN
```

Comment:
Do we really need this `s.AGAIN` - https://github.com/nginx/njs/issues/111#issuecomment-477136307